core/src/main/java/com/graphhopper/routing/AlternativeRoute.java

```diff
@@ -293,10 +285,9 @@ public boolean finished() {
             // For bidir A* and AStarEdge.getWeightOfVisitedPath see comment in AStarBidirection.finished
         }
 
-        public Path searchBest(int to, int from) {
+        public Path searchBest(int from, int to) {
             createAndInitPath();
-            initFrom(to, 0);
-            initTo(from, 0);
+            init(from, 0, to, 0);
```

Comment:
oh, wow...what was this. Thanks!

Reply:
The code was correct, but the variable names were misleading :)